src/Sylius/Bundle/ChannelBundle/Behat/ChannelContext.php

```diff
@@ -17,6 +17,9 @@
 use Sylius\Component\Core\Model\ChannelInterface;
 use Sylius\Component\Core\Model\ProductInterface;
 
+/**
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
```

Comment:
Are you sure that Paweł is the author of this class? The only time one you add an `@author` note is when you add yourself :wink: